File: CS App1/Guitar.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace CS_App1 {
    public class Guitar {
        private string serialNumber;
        private double price;
        private string builder;
        private string model;
        private string type;
        private string backWood;
        private string topWood;

        public Guitar(string serialNumber, double price, string builder, string model, string type, string backWood, string topWood) {
            this.serialNumber = serialNumber;
            this.price = price;
            this.builder = builder;
            this.model = model;
            this.type = type;
            this.backWood = backWood;
            this.topWood = topWood;
        }

        public string getSerialNumber() {
            return this.serialNumber;
        }

        public double getPrice() {
            return this.price;
        }

        public void setPrice(float price) {
            this.price = price;
        }

        public string getBuilder() {
            return this.builder;
        }

        public string getModel() {
            return this.model;
        }

        public string getType() {
            return this.type;
        }

        public string getBackWood() {
            return this.backWood;
        }

        public string getTopWood() {
            return this.topWood;
        }
    }
}

File: VB App1/Guitar.vb
Public Class Guitar
    Private serialNumber As String
    Private price As Double
    Private builder As String
    Private model As String
    Private type As String
    Private backWood As String
    Private topWood As String

    Public Sub New(serialNumber As String, price As Double, builder As String, model As String, type As String, backWood As String, topWood As String)
        Me.serialNumber = serialNumber
        Me.price = price
        Me.builder = builder
        Me.model = model
        Me.type = type
        Me.backWood = backWood
        Me.topWood = topWood
    End Sub

    Public Function getSerialNumber() As String
        Return Me.serialNumber
    End Function

    Public Function getPrice() As Double
        Return Me.price
    End Function

    Public Sub setPrice(price As Single)
        Me.price = price
    End Sub

    Public Function getBuilder() As String
        Return Me.builder
    End Function

    Public Function getModel() As String
        Return Me.model
    End Function

    Public Function getGuitarType() As String
        Return Me.type
    End Function

    Public Function getBackWood() As String
        Return Me.backWood
    End Function

    Public Function getTopWood() As String
        Return Me.topWood
    End Function
End Class
